addons/website/static/src/snippets/s_table_of_content/000.js

```diff
@@ -23,7 +24,8 @@ const TableOfContent = publicWidget.Widget.extend({
      */
     destroy() {
         this.$target.css('top', '');
-        this.$target.find('.s_table_of_content_navbar').css('top', '');
+        this.tableOfContentNavbarEl.style.top = '';
```

Comment:
- In stable versions, I would be more careful: if `s_table_of_content_navbar` does not exist (for whatever custo reason), this line will now crash.
- In stable versions, make a minimal diff: you don't need to share the variable in `start` etc. It's a nice change** for master but I would avoid risking moving code around in stable: especially considering I have to review it and every line of diff in stable fixes is time I waste making sure it is 100% safe and OK. Changing `.css('top', ...)` to `.style.top` is also a change that is not needed in a stable fix: it has 0 benefits to the stable version and only has risks (even if extremely small) => Reserve those changes for master.

** Note: I'd like that to be a nice change but I am actually still not 100% sure that it is perfect in edit mode: I would be worry that the navbar element is somehow reconstructed/remove-then-recreated/whatever by the editor and that you thus don't have the same navbar between the start and the destroy. That should be ok (so until proven it is not, I'd encourage you to do that in **master** code) but it's always a gray area for public widgets that are also executed in edit mode, so definitely do not change that in stable versions.